dll/win32/ieframe/lang/ro-RO.rc

```diff
@@ -42,8 +48,8 @@ IDR_BROWSE_MAIN_MENU MENU
 
 STRINGTABLE
 {
-    IDS_INTERNET "Navigatorul de internet"
-    IDS_INTERNET_DESCRIPTION "Opens a Web browser and displays information on the Internet."
+    IDS_INTERNET "Navigatorul de Internet"
+    IDS_INTERNET_DESCRIPTION "Deschide navigatorul de rețea și afișează informații de pe Internet."
```

Comment:
Although in colloquial speech one may hear "informații de pe Internet", the correct wording is "informații din Internet".